### DISCUSSION TITLE
StatusTag storybook component

### DISCUSSION THREAD

**Discussion by dgateles** [mondaycom/vibe#2405] [Q&A] [answered]
Hello,

I would like to know if I can use this component from your library. I have been looking for something like this for a long time, and while going through your Storybook documentation, I noticed that some of your items have badges for Alpha, Beta, etc.

If it’s possible, how can I go about using it?

I would be immensely grateful!

**Comment by talkor** [comment, marked as answer]
Hey @dgateles! You can definitely use it from the [vibe-storybook-components](https://www.npmjs.com/package/vibe-storybook-components) package. The StatusTag is a part of the SidebarItem. You can import them like this:
```
import { StatusTag, SidebarItem } from "vibe-storybook-components";
```

and use it same as we [use it here](https://github.com/mondaycom/vibe/blob/master/packages/core/.storybook/manager.jsx#L13..L28). Check out also the [docs](https://style.monday.com/storybook-blocks/?path=/docs/components-sidebaritem--docs) for these components

**Comment by dgateles** [comment]
Hello @talkor, thank you very much for allowing the use of your plugin.

I followed the recommended steps, but I wasn’t successful, as shown in the image:

![Captura de Tela 2024-09-02 às 09 48 21](https://github.com/user-attachments/assets/a8bedf96-db79-482d-abdc-76de2868a4e6)

**Comment by talkor** [comment]
@dgateles please share the code of how you used and I'll try to help
